### PR TITLE
NOISSUE - Use HTTP Status in SDK error messages

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

**Review by blokovi — DISMISSED** [review]

File: ws/adapter.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ var (
 	// ErrFailedSubscription indicates that client couldn't subscribe to specified channel.
 	ErrFailedSubscription = errors.New("failed to subscribe to a channel")
 
-	// ErrFailedConnection indicates that service couldn't connect to message broker.
-	ErrFailedConnection = errors.New("failed to connect to message broker")
+	// ErrFailedConnect indicates that service couldn't connect to message broker.
```
Comment (drasko) [inline ws/adapter.go:28]: I prefer `FailedConnection`

Comment (manuio) [reply ws/adapter.go:28]: @drasko Should I use `ErrFailedDisconnection`?

**Review by drasko — APPROVED** [review]
LGTM

File: ws/adapter.go (hunk 1 above)
Comment (drasko) [reply ws/adapter.go:28]: No need to change, this was just neat-picking, and probably not correct. Let's move on, I approved.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**